### PR TITLE
Implement fleet telematics find waypoints sequence here service.

### PR DESCRIPTION
**Pull request by viniciusgabrielfo**
- Add primary integrate with the Fleet Telematics Waypoints Sequence Here service (https://developer.here.com/documentation/routing-waypoints/dev_guide/topics/what-is.html).

### CODE REVIEW

File: here/fleet_telematics.go [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+	fmt.Printf("%v", fleetTelematicsParams)
+	return fleetTelematicsParams
+}
+
+// FindSequence with given parameters.
+func (s *FleetTelematicsService) FindSequence(params *FleetTelematicsParams) (*FleetTelematicsWaypointsSequenceResponse, *http.Response, error) {
+	routeSequence := new(FleetTelematicsWaypointsSequenceResponse)
+	apiError := new(APIError)
+	resp, err := s.sling.New().Get("findsequence.json").QueryStruct(params).Receive(routeSequence, apiError)
+	fmt.Printf("\n%v\n", routeSequence)
```
Comment (abdullahselek) [inline here/fleet_telematics.go]: Please remove lines that prints sequence and response.

File: here/resources/fleet_telematics_find_sequence_response.json [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+                "waiting": 0
+            }
+        }
+    ],
+    "errors": [],
+    "processingTimeDesc": "921ms",
+    "responseCode": "200",
+    "warnings": null,
+    "requestId": null
+}
```
Comment (abdullahselek) [inline here/resources/fleet_telematics_find_sequence_response.json]: Please add a new empty line in the end.

**Review by abdullahselek — CHANGES_REQUESTED** [review]
Hi @ViniciusGabrielf thanks for your PR! I added a few minor comments please have a look.

File: here/fleet_telematics.go (hunk 1 above)
Comment (viniciusgabrielfo) [reply here/fleet_telematics.go]: Removed. Sorry, I was using it for debugging and forgot to remove it.

File: here/resources/fleet_telematics_find_sequence_response.json (hunk 2 above)
Comment (viniciusgabrielfo) [reply here/resources/fleet_telematics_find_sequence_response.json]: Can you explain to me why there must be a blank line? Already added.

Comment (abdullahselek) [reply here/resources/fleet_telematics_find_sequence_response.json]: Thanks for the changes 👍 There are different opinions about that I can suggest you to read the comments on https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file.

**Pull request merged**